Warsaw_Tram/Extensions/UIViewController.swift

```diff
@@ -1,37 +1,33 @@
 //
-//  AlertHelper.swift
+//  UIViewController.swift
 //  Warsaw_Tram
 //
-//  Created by Małgorzata Dziubich on 25/06/16.
+//  Created by Małgorzata Dziubich on 26/06/16.
 //  Copyright © 2016 Małgorzata Dziubich. All rights reserved.
 //
 
 import UIKit
 
-protocol AlertHelperProtocol {
-    func showAlert(title: String, message: String, okAction: UIAlertAction, cancelAction: UIAlertAction, presentingViewController: UIViewController)
-    func showAlert(title: String, message: String, okAction: UIAlertAction, presentingViewController: UIViewController)
-}
+extension UIViewController {
 
-class AlertHelper: NSObject, AlertHelperProtocol {
-    func showAlert(title: String, message: String, okAction: UIAlertAction, cancelAction: UIAlertAction, presentingViewController: UIViewController) {
+    func showAlert(title: String, message: String, okAction: UIAlertAction, cancelAction: UIAlertAction) {
         // Create the alert controller
         let alertController = UIAlertController(title: title, message: message, preferredStyle: .Alert)
-
+        
         // Add the actions
         alertController.addAction(okAction)
         alertController.addAction(cancelAction)
         
         // Present the controller
-        presentingViewController.presentViewController(alertController, animated: true, completion: nil)
+        self.presentViewController(alertController, animated: true, completion: nil)
```

Comment:
Don't use `self` when is it not necessary. This is not a mistake, just good practice.